components/newfoldercomponent/ParagraphComponent.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div  contenteditable="true"></div>
+    <div  contenteditable="true">Hello world</div>
   </div>
 </template>
```

Comment:
Remove the "Hello World" text here, it's not needed. Update this component to accept two props:
- the index that is passed from the page component (You can name it `index`)
- the paragraph that is passed (you can name it `paragraph`)